src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventSchema.scala

```diff
@@ -0,0 +1,173 @@
+package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra
+
+import java.util.Date
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.FortisEvent
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.dto._
+import java.text.Collator
+import java.util.Locale
+
+import com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries.{Period, PeriodType}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.gender.GenderDetector.{Female, Male}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations._
+
+object CassandraEventSchema {
+  def apply(item: FortisEvent, batchid: String): Event = {
+    Event(
+      pipelinekey = item.details.pipelinekey,
+      externalsourceid = item.details.externalsourceid,
+      computedfeatures = Utils.getFeature(item),
+      eventtime = item.details.eventtime,
+      batchid = batchid,
+      topics = item.analysis.keywords.map(_.name.toLowerCase),
+      fulltext = s"[${Option(item.details.title).getOrElse("")}] - ${item.details.body}",
+      placeids = item.analysis.locations.map(_.wofId),
+      eventlangcode = item.analysis.language.orNull,
+      eventid = item.details.eventid,
+      insertiontime = new Date().getTime,
+      body = item.details.body,
+      sourceurl = item.details.sourceurl,
+      title = item.details.title)
+  }
+}
+
+object CassandraPopularPlaces {
+  def apply(item: Event): Seq[PopularPlaceAggregate] = {
+    for {
+      kw <- Utils.getConjunctiveTopics(Option(item.computedfeatures.keywords))
+      location <- item.computedfeatures.places
+      periodType <- Utils.getCassandraPeriodTypes
+    } yield PopularPlaceAggregate(
+      pipelinekey = item.pipelinekey,
+      centroidlat = location.centroidlat,
+      centroidlon = location.centroidlon,
+      placeid = location.placeid,
+      periodstartdate = Period(item.eventtime, periodType).startTime(),
+      periodenddate = Period(item.eventtime, periodType).endTime(),
+      periodtype = periodType.periodTypeName,
+      period = periodType.format(item.eventtime),
+      externalsourceid = item.externalsourceid,
+      mentioncount = item.computedfeatures.mentions,
+      conjunctiontopic1 = kw._1,
+      conjunctiontopic2 = kw._2,
+      conjunctiontopic3 = kw._3,
+      avgsentimentnumerator = 0,
```

Comment:
Given that we already have one sentiment in the numerator, should the denominator be 1?